project/frontend/src/components/SGonksLists/ShortSGonksList/ShortSGonksRow/ShortSGonksRow.js

```diff
@@ -2,11 +2,14 @@ import React from "react";
 import classes from "./ShortSGonksRow.module.css";
 
 const ShortSGonksRow = (props) => {
+  console.log(props);
   const getDiffIndicator = () => {
     const diff =
-      props.sgonk.currentPrice > props.sgonk.priceYesterday
+      props.datapoints[props.datapoints.length - 1] >
+      props.datapoints[props.datapoints.length - 2]
```

Comment:
{nit} may be more readable if you do priceDelta = props.datapoints[props.datapoints.length - 1]  -  props.datapoints[props.datapoints.length - 2]
and then priceDelta < 0 ? ..  : priceDelta > 0 ? ... : ...

Reply:
Can do

Reply:
Done